agnostic-svelte/src/lib/components/Pagination/Pagination.svelte

```diff
@@ -123,6 +124,15 @@
     next: "Next",
   };
 
+  function genPages(page) {
+    if (pageGenerator) {
```

Comment:
We set this to a default `usePagination` so I'm thinking we don't need the guard conditional no?

Reply:
this is if someone were to set the pageGenerator to undefined and update the pages property manually for some reason

Reply:
I think we should probably throw an error because that seems like a bug in there code. Or am I missing a legitimate use case?